[2024-05-01T10:21:07.000+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-01T10:21:07.015+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final_exam.starting manual__2024-05-01T10:21:05.315844+00:00 [queued]>
[2024-05-01T10:21:07.020+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final_exam.starting manual__2024-05-01T10:21:05.315844+00:00 [queued]>
[2024-05-01T10:21:07.020+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-01T10:21:07.032+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): starting> on 2024-05-01 10:21:05.315844+00:00
[2024-05-01T10:21:07.037+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=67) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-01T10:21:07.038+0000] {standard_task_runner.py:63} INFO - Started process 69 to run task
[2024-05-01T10:21:07.038+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final_exam', 'starting', 'manual__2024-05-01T10:21:05.315844+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/Project.py', '--cfg-path', '/tmp/tmpjbtr4kg4']
[2024-05-01T10:21:07.039+0000] {standard_task_runner.py:91} INFO - Job 31: Subtask starting
[2024-05-01T10:21:07.071+0000] {task_command.py:426} INFO - Running <TaskInstance: Final_exam.starting manual__2024-05-01T10:21:05.315844+00:00 [running]> on host 73ada3aeb297
[2024-05-01T10:21:07.132+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final_exam' AIRFLOW_CTX_TASK_ID='starting' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T10:21:05.315844+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-01T10:21:05.315844+00:00'
[2024-05-01T10:21:07.133+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-01T10:21:07.133+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-01T10:21:07.134+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Chuẩn bị cào dữ liệu"']
[2024-05-01T10:21:07.141+0000] {subprocess.py:86} INFO - Output:
[2024-05-01T10:21:07.142+0000] {subprocess.py:93} INFO - Chuẩn bị cào dữ liệu
[2024-05-01T10:21:07.143+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-01T10:21:07.143+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-01T10:21:07.162+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Final_exam, task_id=starting, execution_date=20240501T102105, start_date=20240501T102107, end_date=20240501T102107
[2024-05-01T10:21:07.212+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-01T10:21:07.230+0000] {taskinstance.py:3482} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-05-01T10:21:07.233+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
